pyipmi/msgs/constants.py

```diff
@@ -383,6 +385,48 @@
         0x86: 'requested maximum privilege level exceeds user and/or channel'
               ' privilege limit',
     },
+    (NETFN_APP | 1, CMDID_SET_SESSION_PRIVILEGE_LEVEL, None): {
```

Comment:
I choose to set `NefFn | 1` as the key in the description dictionary rather than just `NetFn` because we use the dictionary only for IPMI Responses, but the choice can be debated.